Marlin/src/feature/spindle_laser.h

```diff
@@ -49,11 +58,11 @@ class SpindleLaser {
 
   static void init();
 
-  static inline bool enabled() { return !!power; }
+  static inline bool enabled() { return power == 0; }
```

Comment:
Is it true that `power == 0` makes it enabled? That is the opposite of the previous code, `!!power`.

Reply:
Yep; I noticed that while testing

It should be `power >0` instead

Reply:
Fixed by jediminer543@fced0ce72e8299155a8a626db8d7c41eee18c098